paimon-core/src/main/java/org/apache/paimon/schema/SchemaValidation.java

```diff
@@ -107,7 +109,7 @@ public static void validateTableSchema(TableSchema schema) {
                             checkState(
                                     !SYSTEM_FIELD_NAMES.contains(f),
                                     String.format(
-                                            "Field name[%s] in schema cannot be exist in [%s]",
+                                            "Field name[%s] in schema cannot be exist in %s",
```

Comment:
Why need to change the info of `SYSTEM_FIELD_NAMES`? IMO, it's no need to change.

Reply:
I just modified the printed exception information, 
before modification: `Field name["_VALUE_COUNT"] in schema cannot be exist in [["_VALUE_COUNT", "_SEQUENCE_NUMBER", "_VALUE_KIND"]]` ,
 after modification: `Field name["_VALUE_COUNT"] in schema cannot be exist in ["_VALUE_COUNT", "_SEQUENCE_NUMBER", "_VALUE_KIND"]`